argilla/pyproject.toml

```
@@ -4,7 +4,7 @@ description = "The Argilla python server SDK"
 authors = [
     {name = "Argilla", email = "contact@argilla.io"},
 ]
-requires-python = ">=3.9,<=3.12"
+requires-python = ">= 3.9"
```

Thanks for this @dcfidalgo!!! (and happy to see you here :-)) I think this explicit upper limit also works and avoids installing for unsupported versions.
```suggestion
requires-python = ">= 3.9, <3.13"
```

---

Yes, this also works for me! :) Not sure where the pyarrow installation error comes from, it should support 3.12 ...